code/modules/reagents/machinery/grinder.dm

```diff
@@ -76,9 +76,6 @@
 	SSnano.update_uis(src)
 	return 0
 
-/obj/machinery/reagentgrinder/attack_ai(mob/user as mob)
```

Comment:
does the usability tweak in attackby not work? attack_ai, as the proc name implies, is for AIs, not direct touching.
`//Useability tweak for borgs`
`if (istype(I,/obj/item/gripper))`
`	ui_interact(user)`
`	return`

Reply:
Gripper transfer items just fine, but grinder's interface can't be opened by silicon.

Reply:
and clicking with an empty gripper doesn't work?

Reply:
It doesn't.

Reply:
Regardless, I see no reason why robot can remotely interface with almost any machine, but not an industrial grinder.
Some modules have to interact with it as part of their "job".

Reply:
I thought maybe it'd let them teleport beakers, but from what I saw the beakers and inserted items eject at the location of the grinder. Not sure if there's a button on the UI that lets you teleport stuff in though.

Reply:
If such problem even exist, there is a better way of fixing it than simply locking machine for robots.